libc/dirent/opendir.c

```diff
@@ -0,0 +1,447 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * test/test_strftime.c
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Arkadiusz Kozlowski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <time.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <dirent.h>
+#include <limits.h>
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <string.h>
+
+#include <unity_fixture.h>
+
+#include "common.h"
+
+#define MAIN_DIR "test_opendir"
+
+
+int create_directories(int num_of_dirs, char identifier)
```

Comment:
Identifier is not needed, functions defined by yourself should be static and have `test_` prefix